docs/apps/lnw/es2k/es2k-linux-networking-lag.md

```diff
@@ -261,7 +261,8 @@ The following configuration assumes that:
 - Underlay IDPF netdev has a VSI value 11 for second LAG member
 
 ```bash
-ip link add bond0 type bond miimon 100 mode active-backup
+For static LAG: ip link add bond0 type bond miimon 100 mode active-backup
```

Comment:
The text within a code block should be valid code in the specified language. There should be a clear distinction between the commands or instructions to be executed and any text you've added to explain. The rule of thumb is: _the reader should be able to copy the text within the code block and paste it directly into their editor or shell_.

One way of doing this is to move the explanatory text outside the code block.

For static LAG:
```code
ip link add bond0 type bond miimon 100 mode active-backup
```
Another is to make the explanatory text a _comment_:
```suggestion
# For static LAG: ip link add bond0 type bond miimon 100 mode active-backup
```

Given the structure of the information you're presenting, I would suggest making the two explanatory lines comments:
```code
# For static LAG: ip link add bond0 type bond miimon 100 mode active-backup
# For dynamic LAG: ip link add bond0 type bond miimon 100 mode 802.3ad
ip link set <IDPF netdev for VSI 10> down
ip link set <IDPF netdev for VSI 10> master bond0
ip link set <IDPF netdev for VSI 11> down
```
Notice that GitHub includes a "copy" button to make it easier for the reader to use the example directly.